src/js/components/point-card/flag-point-card.js

```diff
@@ -0,0 +1,75 @@
+/*eslint-disable no-unused-vars*/
+import React from 'react';
+import { Card, CardActions, CardText, FlatButton, Paper, TextField, List, ListItem } from 'material-ui';
+
+import PointCard from './point-card';
+import { FormBlock } from '../block';
+import RatingSelector from '../rating-selector';
+/*eslint-enable no-unused-vars*/
+
+import { bindAll, cloneDeep } from 'lodash';
+import history from '../../history';
+
+import uuid from 'node-uuid';
+import { Service } from 'btc-models';
+
+
+
+export class FlagPointCard extends PointCard {
+ 
+  constructor( props ) {
+    super( props );
+    bindAll( this, 'callToFlagPoint' );
+    this.errorMessage = "";
+  }
+
+  getCardState() {
+    return 'point-card--rate';
+  }
+
+  getCardAction() {
+    const goBack = history.goBack.bind( history );
+    return <FlatButton label="Go Back"
+             onTouchTap={ goBack } />;
+  }
+
+  onCommentUpdate( event ) {
+    const {setComment} = this.props;
+    const comment = event.target.value;
+    setComment( comment );
+  }
+  getCardContent() {
+    return (
+      <div className="point-card__content">
+        { this.getOtherEntry() }
+      </div>
+      );
+  }
+
+callToFlagPoint( values ) {
+    const {flagPoint} = this.props;
+    flagPoint( this.point._id, values.reason );
+  }
+
+  getOtherEntry() {
```

Comment:
Should this be called "other" any more now that the open comment field is the only option?